app/controllers/catarse_pagarme/credit_cards_controller.rb

```diff
@@ -12,6 +12,13 @@ def create
       render json: { payment_status: 'failed', message: e.message }
     end
 
+    def get_installment_json
+      render json: installments_for_json.to_json
+    end
+
+    def get_saved_cards
+      render json: current_user.credit_cards.to_json
```

Comment:
@vicnicius we already have in catarse app a section for handle with user credit cards.